f2/server.js

```diff
@@ -87,8 +93,15 @@ app.get('/', function(req, res, next) {
         : 'http://www.antifraudcentre-centreantifraude.ca/report-signalez-eng.htm',
     )
   } else {
-    availableData.numberOfRequests += 1
-    availableData.lastRequested = new Date()
+    var referrer = req.headers.referer
```

Comment:
should this be `req.headers.referrer`?

Reply:
Probably, but it works??

https://en.wikipedia.org/wiki/HTTP_referer

"The HTTP referer **(a misspelling of referrer[1])** is an optional HTTP header field that identifies the address of the webpage (i.e. the URI or IRI) which is linked to the resource being requested. By checking the referrer, the new webpage can see where the request originated."

Reply:
for #@%@#$ sake.